region/wreckedship/main/Wrecked Ship East Super Room.json

```diff
@@ -523,6 +523,21 @@
         "Note that this strat is useless in vanilla, as the item only spawns if Phantoon has been killed."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "WS East Supers Robot Clip, Use Flash Suit",
+      "requires": [
+        "canMidAirMorph",
+        "canKago",
+        "canHorizontalShinespark",
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 1, "excessFrames": 1}}
+      ],
+      "note": [
+        "Kago into the worker robot and shinespark horizontally into the wall in order to break the bomb block with minimal Energy loss."
```

Comment:
Interesting that this has to be a lowish horizontal shinespark -- not from a crouch jump.  Or the robot pushes you up too high.